src/infinite-list/index.tsx

```diff
@@ -137,17 +137,14 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
 
   const onMomentumScrollEnd = useCallback(
     event => {
-      if (pageIndex.current) {
```

Comment:
We can just check if isUndefined(pageIndex.current) instead of removing the condition